### ISSUE TITLE
Creation modal warnings (same for ingredient, recipe, category and ustensil)

### ISSUE THREAD

**Issue by marquito73** [lviardcretat/recepto-old#18]
Warning: DialogContent requires a DialogTitle for the component to be accessible for screen reader users.

Warning: Missing Description or aria-describedby="undefined" for DialogContent.
